packages/features/audit-logs/lib/getAuditLogManager.ts

```diff
@@ -0,0 +1,47 @@
+import type { AuditLogsManager } from "@calcom/features/audit-logs/types";
+import logger from "@calcom/lib/logger";
+import { getPiiFreeCredential } from "@calcom/lib/piiFreeData";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import type { Credential } from "@calcom/prisma/client";
+
+const auditLogImplementations = {
```

Comment:
Wanted to create this automatically, kinda like how all generated files are created but since so very few apps are using this i thought it would be easier to manually edit the object.  If a new integration is developed but not defined in this object, the integration's AuditLogManager will not be found and our genericImplementation will be used. (reporting events is just a console.log)